File: CompilationTest/StructureTest1/results/StructureTest1.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

class MyList<T> {
    public List<T> items;
}

class Item {
    public Integer offset = 5;
    public String strTest = "test" + "test2";
    public String strConstr = "constr";

    public Item(String strConstr) throws Exception {
        this.strConstr = strConstr;
    }
}

class Container {
    public MyList<Item> itemList;
    public MyList<String> stringList;

    public void method0() throws Exception
    {
    }
    
    public String method1(String str) throws Exception
    {
        return str;
    }
}

class Program {
    public static void main(String[] args) throws Exception {
        System.out.println("ok");
    }
}

File: CompilationTest/StructureTest1/results/StructureTest1.cs
using System;
using System.Collections.Generic;

public class MyList<T>
{
    public List<T> Items;
}

public class Item
{
    public int Offset = 5;
    public string StrTest = "test" + "test2";
    public string StrConstr = "constr";

    public Item(string strConstr)
    {
        this.StrConstr = strConstr;
    }
}

public class Container
{
    public MyList<Item> ItemList;
    public MyList<string> StringList;

    public void Method0()
    {
    }
    
    public string Method1(string str)
    {
        return str;
    }
}

public class Program
{
    static public void Main(string[] args)
    {
        Console.WriteLine("ok");
    }
}